test/unit/TestNXOSDriver.py

```diff
@@ -71,10 +73,12 @@ def show(self, command, fmat = 'xml', text = False):
             extension = 'txt'
 
         filename = re.sub(r'[\[\]\*\^\+\s\|\/]', '_', command)
-        mock_file = 'nxos/mock_data/{filename}.{extension}'.format(
+        mock_file = '{filename}.{extension}'.format(
             filename  = filename[0:150],
             extension = extension
         )
+        curr_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
```

Comment:
Honestly, it may.  This is the way I've always done it (seen on various blogs and stackoverflow), but it looks like __file__ could do as well.  David B asked a similar question in another PR.  Can change it to that if you'd like, reply w/ "Make it so" and it will be made so.